catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js

```diff
@@ -140,77 +159,101 @@ function (ich,Marionette,Backbone,ConfigurationEdit,Service,Utils,wreqr,_,$,moda
                 configs.add(currentService);
             }
             if (!_.isUndefined(disabledConfigs)) {
-                disabledConfigs.each(function(config) {
+                disabledConfigs.each(function (config) {
                     configs.add(config.get('service'));
                 });
             }
             return configs;
         },
+        getActions: function (action) {
+            var disabledConfigs = this.model.get('disabledConfigurations');
+            var currentConfig = this.model.get('currentConfiguration');
+            var actions;
+            if (!_.isUndefined(currentConfig)) {
+                actions = currentConfig.get(action);
+            }
+            if (!_.isUndefined(disabledConfigs)) {
+                var actionConfig = _.find(disabledConfigs.models, function (config) {
```

Comment:
This isn't the right way to do this. You can just do this.model.get('disabledConfigurations').find(...)